haskell/test/Lorentz/Contracts/Test/Management.hs

```diff
@@ -97,13 +97,13 @@ managementSpec originate = do
           (constructTransfersFromSender (#from_ .! wallet1) [])
         mgmXtzReceived err
 
-    it "token metadata registry is present in storage" $ integrationalTestExpectation $ do
-      mrAddress <- originateMetadataRegistry
-      let originationParams = defaultOriginationParams { opTokenMetadataRegistry = Just mrAddress }
-      withOriginated originate originationParams $ \stablecoinContract -> do
-        lExpectStorage @Storage stablecoinContract $ \storage ->
-          unless (sTokenMetadataRegistry storage == mrAddress) $
-            Left $ CustomTestError "Malformed token metadata registry address in contract storage"
+    -- it "token metadata registry is present in storage" $ integrationalTestExpectation $ do
+    --   mrAddress <- originateMetadataRegistry
+    --   let originationParams = defaultOriginationParams { opTokenMetadataRegistry = Just mrAddress }
+    --   withOriginated originate originationParams $ \stablecoinContract -> do
+    --     lExpectStorage @Storage stablecoinContract $ \storage ->
+    --       unless (sTokenMetadataRegistry storage == mrAddress) $
+    --         Left $ CustomTestError "Malformed token metadata registry address in contract storage"
```

Comment:
Why not fully remove it? It seems to be specific of the registry that no longer exists.

Reply:
Removed